dandiapi/settings/_sentry_utils.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from sentry_sdk._types import SamplingContext
+
+
+def get_sentry_performance_sample_rate(sampling_context: SamplingContext) -> float:
```

Comment:
It seems this function still exists in `apps.py`

Reply:
It should have been removed there and moved here. @mvandenburgh